pkgs/applications/audio/psst/default.nix

```diff
@@ -1,4 +1,18 @@
-{ lib, fetchFromGitHub, rustPlatform, alsa-lib, atk, cairo, dbus, gdk-pixbuf, glib, gtk3, pango, pkg-config, makeDesktopItem }:
```

Comment:
Please do not include formatting changes in this PR.

Keep only the new fetcher related changes.